tests/app/broadcast_areas/test_models.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from app.broadcast_areas.models import CustomBroadcastArea
+from tests.app.broadcast_areas.custom_polygons import BRISTOL, SANTA_A, SKYE
+
+
+@pytest.mark.parametrize(('simple_polygon', 'expected_wards_length'), [
+    (SKYE, 2),
+    (BRISTOL, 12),
+    (SANTA_A, 0)  # does not overlap with UK
+])
+def test_custom_broadcast_area_overlapping_electoral_wards(
+    simple_polygon,
+    expected_wards_length,
+):
+    custom_area = CustomBroadcastArea(
+        name='foo', polygons=[simple_polygon]
+    )
+
+    assert len([
+        area.name for area in
+        custom_area.overlapping_electoral_wards
+    ]) == expected_wards_length
```

Comment:
The list comprehension isn’t adding anything here – this assertion is equivalent:
```suggestion
    assert len(custom_area.overlapping_electoral_wards) == expected_wards_length
```

Reply:
Good spot! I've amended the first commit with that.